using-mobile-extensions/adobe-target-vec.md

```diff
@@ -0,0 +1,144 @@
+# Adobe Target - VEC
+This extension enables Target VEC (Visual Experience Composer) functionality. The Target VEC for Native Mobile Apps lets you create activities and personalize content on native mobile apps in a do-it-yourself fashion without continuous development dependencies and app-release cycles.
```

Comment:
Please reword as follows:

"This extension enables the Adobe Target Visual Experience Composer (Target VEC)."